public class ArithmeticExceptionExample {
    public static void main(String[] args) {
        try {
            // Divided by Zero
            int result = 10 / 0;
            System.out.println("Result: " + result);
        } catch (ArithmeticException e) {
            System.out.println("Arithmetic Exception: " + e.getMessage());
        }
        
        try {
            // Logarithm of negative or Zero
            double result = Math.log(-1);
            System.out.println("Result: " + result);
        } catch (ArithmeticException e) {
            System.out.println("Arithmetic Exception: " + e.getMessage());
        }
        
        try {
            // Tan 90 Degree
            double result = Math.tan(Math.toRadians(90));
            System.out.println("Result: " + result);
        } catch (ArithmeticException e) {
            System.out.println("Arithmetic Exception: " + e.getMessage());
        }
        
        try {
            // Zero power Zero
            double result = Math.pow(0, 0);
            System.out.println("Result: " + result);
        } catch (ArithmeticException e) {
            System.out.println("Arithmetic Exception: " + e.getMessage());
        }
    }
}
